scripts/clean_landingzone.sql

```diff
@@ -0,0 +1,3 @@
+-- Remove all of the data from all tables; leave the schema intact.
+TRUNCATE landingzone CASCADE;
```

Comment:
Is this a development convenience or is it used somewhere in the code?

Reply:
it is convenience script, similar to workspace and stairway table clean up scripts.